x-pack/plugin/core/build.gradle

```diff
@@ -149,16 +149,8 @@ restResources {
   }
 }
 
-testClusters.configureEach {
-  testDistribution = 'default'
-  setting 'xpack.security.enabled', 'true'
-  setting 'xpack.license.self_generated.type', 'trial'
-  //disabling ILM history as it disturbs testDSXpackUsage test
-  setting 'indices.lifecycle.history_index_enabled', 'false'
-  keystore 'bootstrap.password', 'x-pack-test-password'
-  user username: "x_pack_rest_user", password: "x-pack-test-password"
-  requiresFeature 'es.failure_store_feature_flag_enabled', Version.fromString("8.15.0")
-  systemProperty 'es.queryable_built_in_roles_enabled', 'false'
+tasks.named("javaRestTest") {
+  usesDefaultDistribution("uses the _xpack api")
```

Comment:
From what I can tell this tries to resolve pretty much every x-pack module so I gave up on going down this rabbit hole. Fortunately, these tests suites are pretty light weight so it's not a big deal.